cfgov/cal/views.py

```diff
@@ -122,8 +181,16 @@ def display(request, pdf=False):
             'range_end':range_end,
         })
 
-        if pdf and PDFreactor:
-            return pdf_response(request, context)
+        if pdf:
+            template_name = 'about-us/the-bureau/leadership-calendar/print/index.html'
+            if PDFreactor:
+                return pdf_response(request, context)
+
+    elif form_is_vaild == False and pdf:
+        for error in form.errors.itervalues():
+            messages.error(request, str(error), extra_tags='leadership-calendar')
```

Comment:
Using `str` here doesn't properly handle unicode values in the inputs. For example, try putting `ë` as one of the date fields in the PDF download form. This will raise an unhandled exception and cause a 500 error.